adding_species_manuscript.tex

```diff
@@ -317,82 +320,63 @@ \section*{The utility of \stdpopsim for genome-wide simulations}
+
+%As a result, while the simulations themselves may require considerable computational resources,
+%the most time-consuming and error-prone part of population genomic simulation is
+%often the task of correctly parameterizing simulation software.
 
-% \paragraph*{\Stdpopsim streamlines the parameterization of population genomic simulations}
 The main objective of \stdpopsim is to streamline this process,
-making it less time consuming, less error prone, and more reproducible.
-Contributors use a template to build the model for their species of interest,
-including the required parameter values. This model then goes through a
-vital peer-review process, including validating the choices of parameter values.
-% Elise: Commented out the below sentences because this seems like too much detail here. Replaced with above sentence.
-% To ensure reliability,
-% after a contributor adds a new simulation model to the catalog, it is flagged for review.
-% Then, another contributor independently creates a simulation model
-% based on the same literature sources and the documentation provided by the initial contributor.
-% The two separate models are then compared to each other by automated scripts.
-% Any discrepancies are resolved by the two contributors,
-Any discrepancies are resolved between the contributor and reviewer,
-if necessary with input of additional members of the community.
+and to make it more robust and more reproducible.
+Contributors collect parameter values for their species of interest from the literature,
+and then specify these parameters in a template file for the new model.
+%Ilan: I added some of the details commented out by Elise but streamlined it in one sentence
+This model then goes through a vital peer-review process,
+which involves recreating the model based on the provided documentation,
+and executing automated scripts to compare the two models.
+If discrepancies are found in this process, they are resolved by discussion between the contributor and reviewer,
+and if necessary with input of additional members of the community.
 This quality control process quite often finds subtle bugs \citep[e.g., as in][]{Ragsdale2020}
 or highlights parts of the model that are ambiguously defined by the literature sources.
-This considerably increases the reliability of the resulting simulations in any downstream analysis.
-
-
-% \paragraph*{\textbf{Species-specific genomic architecture is challenging to model}}
-The goal of complete chromosome simulation is important for a number of reasons.
-The organization of genes on chromosomes is a key feature of a species' genome,
-and one that has largely been ignored in population genomic simulation (see \cite{schrider2020background} for a notable exception).
+Importantly, this increases the reliability of the resulting simulations in any downstream analysis.
+
+Another central goal of \stdpopsim is to promote whole-genome simulations,
```

Comment:
s/central/important ?

Reply:
I think i wanted to avoid important because it was used in the previous sentence. Will think of a way to bypass this.

Reply:
To be honest, I'm not a fan of "importantly" as a modifier in general, but that feels like nitpicking. ;-) We could just remove it, ie. "This increases the reliability..." and use "important" in place of "central" in the next sentence.

Reply:
I like Elise's idea

Reply:
This paragraph has two other 'important's in it. I'll use my thesaurus to replace one of them ;-)